core/src/main/resources/i18n/displayStrings.properties

```diff
@@ -1274,6 +1275,19 @@ support.initialInfo=Please enter a description of your problem in the text field
 \t● You need to cooperate with the {2} and provide the information they request to make your case.\n\
 \t● You accepted the rules outlined in the dispute document in the user agreement when you first started the application.\n\n\
 You can read more about the dispute process at: {3}
+support.initialInfoRefundAgent=Please enter a description of why you opened arbitration. Add as much information as possible to speed up dispute resolution time. Mediation and trading chats are not shared with the arbitrator.\n\n\
```

Comment:
=Please describe why you opened arbitration.

Reply:
Change made, screenshot updated above in OP.